src/main/java/org/jenkinsci/plugins/ghprb/GhprbRootAction.java

```diff
@@ -86,13 +87,15 @@ public void doIndex(StaplerRequest req, StaplerResponse resp) {
 
                 for (GhprbRepository repo : getRepos(issueComment.getRepository())) {
                     logger.log(Level.INFO, "Checking issue comment ''{0}'' for repo {1}", new Object[] {issueComment.getComment(), repo.getName()});
-                    repo.onIssueCommentHook(issueComment);
+                    if(repo.checkSignature(body, signature))
```

Comment:
This is going to be an issue when there are multiple github connections.  Maybe this functionality needs to be pushed down to the trigger level.